docs/sections/user_guide/cli/drivers/ioda/help.out

```diff
@@ -0,0 +1,28 @@
+usage: uw ioda [-h] [--version] TASK ...
+
+Execute ioda tasks
+
+Optional arguments:
+  -h, --help
+      Show help and exit
+  --version
+      Show version info and exit
+
+Positional arguments:
+  TASK
+    configuration_file
+      The JEDI YAML configuration file
+    files_copied
+      Files copied for run
+    files_linked
+      Files linked for run
+    provisioned_run_directory
+      Run directory provisioned with all required content
+    run
+      A run
+    runscript
+      The runscript
+    validate
+      Validate the UW driver config
+    validate_only
+      Validate JEDI config YAML
```

Comment:
This is the one thing I'm uncertain about with this PR. The `--validate-only` flag doesn't seem to be enabled with the IODA executable I tested with. It just ran the component. I didn't remove the task, and I'll mention more on that below.